Misc/NEWS.d/next/Build/2021-02-02-16-26-44.bpo-43103.VWeyP_.rst

```diff
@@ -0,0 +1,4 @@
+By default, Python no longer builds ``libpythonMAJOR.MINOR.a`` static library
+and no longer installs it and ``python.o``. These files are only useful to
+build a static Python executable which is an uncommon use case. Add configure
+``--without-static-python`` option to build and install these files.
```

Comment:
Was that meant to be ``--with-static-python`` ?

Reply:
Oh, the NEWS entry was outdated. I wrote the PR and the NEWS entry. Is it better now?

Reply:
if static install is enabled by default now, then yes. thanks!